Sources/Library/Math.swift

```diff
@@ -0,0 +1,154 @@
+//
+//  Math.swift
+//  FirebladeECSDemo
+//
+//  Created by Igor Kravchenko on 29.10.2020.
+//
+
+import FirebladeMath
+
+// swiftlint:disable prefixed_toplevel_constant
+public let pow: (Double, Double) -> Double = FirebladeMath.pow
+public let sqrt: (Double) -> Double = FirebladeMath.sqrt
+public let cos: (Double) -> Double = FirebladeMath.cos
+public let sin: (Double) -> Double = FirebladeMath.sin
+public let ceil: (Double) -> Double = FirebladeMath.ceil
+public let floor: (Double) -> Double = FirebladeMath.floor
+// swiftlint:enable prefixed_toplevel_constant
+public typealias Vector = Vec2d
+public typealias Rect = FirebladeMath.Rect<Double>
+
+public struct Matrix {
```

Comment:
It's totally fine that you define your own implementation of a matrix 3x3, but I'm curious why you did not use the FirebladeMath implementation of it, which for this would be `Mat3d` I guess.

Reply:
I waned to use Mat3x3d initially, but I suppose I was not able to find an overload for
```multiply(_ lhs: Mat3x3d, _ rhs: Mat3x3d)```.

Reply:
I think I can have another try. 
Should I add ```multiply(_ lhs: Mat3x3d, _ rhs: Mat3x3d)``` in AsteroidsGameLibrary?

Reply:
No it's totally fine - don't do the extra work if you do not have the time - there may very well be no multiply overload for now there. If it's actually missing this is something you could either add to FirebladeMath package or at least let me know so I can add that myself. 
There should in theory be a `multiply(_ lhs: Mat3x3d, _ rhs: Mat3x3d)` and an operator `operator * (_ lhs: Mat3x3d, _ rhs: Mat3x3d)` but this might be incomplete for now, because I currently use mostly the `Float` variants.

Reply:
Let's leave it as it is for now then.